docs-src/concepts/what-is-a-schedule.md

```diff
@@ -28,6 +28,12 @@ Schedules provide a more flexible and user-friendly approach than [Temporal Cron
 A Schedule has an identity and is independent of a Workflow Execution.
 This differs from a Temporal Cron Job, which relies on a cron schedule as a property of the Workflow Execution.
 
+:::caution
+
+If the result returned by the scheduled workflow exceeds max block size minus temporal extra information (2MB - 1KB) this result will not be available for the next run.
```

Comment:
```suggestion
If the result returned by the scheduled Workflow Execution exceeds max block size minus temporal extra information (2MB - 1KB) this result will not be available for the next run.
```

What is "temporal extra information"?

Reply:
@flossypurse Thank you for the comment. Temporal wraps results produced by the scheduled workflow in an envelope with some extra information which allows it to find the result and provide it to the next run of the workflow.  Do you think we should explain it here?

Reply:
Well, either remove the mention of the "extra information" completely - because the developer doesn't actually need to worry about that, or yes specify what we mean by that - perhaps something like:

> If the result returned by the scheduled Workflow Execution exceeds max block size (2MB - 1KB) this result will not be available for the next run. The max block size is not affected by the extra meta data that Temporal needs to identify the result and provide it to the next Workflow Execution.